pyproject.toml

```diff
@@ -20,6 +20,7 @@ mypy = "1.9.0"
 ruff = "0.2.1"
 pytest = "8.0.0"
 pytest-html = "4.1.1"
+requests = "^2.32.3"
```

Comment:
Do we need this? I don't see `requests` being used anywhere

Reply:
Ah I forgot to remove. Removed